README.md

```diff
@@ -229,7 +229,7 @@ or on the command line:
 
 	<installdir>/bin/janus --help
 	
-	janus 0.2.0
```

Comment:
Seeing as libcurl is required to build the event handler, we should probably note that in the readme. On Ubuntu I am using libcurl4-openssl-dev